src/screens/HomeScreen.js

```diff
@@ -63,6 +64,27 @@ export const HomeScreen = ({ navigation }) => {
   } = sections;
   const [refreshing, setRefreshing] = useState(false);
 
+  const interactionHandler = useCallback((response) => {
+    const data = response?.notification?.request?.content?.data;
+    const queryType = data?.query_type 
+      ? getQueryType(data.query_type) 
+      : undefined;
+
+    if (data?.id 
+      && queryType
```

Comment:
this looks weirdly formatted, should be `if (data?.id && queryType)` in one line, right? is eslint not complaining?

Reply:
I think this is an artefact from having more conditions there. I changed it to be in one line in e2f9b849f1157cec42932f4fc81cf4ce13f20fc8